docs/Makefile

```diff
@@ -8,19 +8,24 @@ SPHINXBUILD   ?= uv run sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 
+.PHONY: help clean rust-html html uv-sync
+
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
```

Comment:
Makefile shouldn't be PHONY. We want to rebuild all targets if the Makefile changes.